test/cmdlineTests/optimizer_array_sload/output

```diff
@@ -31,27 +31,27 @@ object "Arraysum_34" {
                         if callvalue() { revert(_1, _1) }
                         if slt(add(calldatasize(), not(3)), _1) { revert(_1, _1) }
                         let var_sum := _1
-                        /// @src 0:368:378
-                        let var_i := /** @src 0:80:429 */ _1
+                        /// @src 0:368:378  uint i = 0
+                        let var_i := /** @src 0:80:429  contract Arraysum {... */ _1
                         let _2 := sload(_1)
-                        /// @src 0:364:423
+                        /// @src 0:364:423  for(uint i = 0; i < values.length; i++)...
                         for { }
-                        /** @src 0:380:397 */ lt(var_i, _2)
-                        /// @src 0:368:378
+                        /** @src 0:380:397  i < values.length */ lt(var_i, _2)
+                        /// @src 0:368:378  uint i = 0
                         {
-                            /// @src 0:80:429
+                            /// @src 0:80:429  contract Arraysum {...
                             if eq(var_i, not(0)) { panic_error_0x11() }
-                            /// @src 0:399:402
-                            var_i := /** @src 0:80:429 */ add(var_i, 1)
+                            /// @src 0:399:402  i++
+                            var_i := /** @src 0:80:429  contract Arraysum {... */ add(var_i, 1)
```

Comment:
These inline comments are probably not very useful when they point at the whole contract. Maybe we should have a heuristic that cuts them out?